webv2/utilities/utilities.go

```diff
@@ -236,15 +246,38 @@ func RemovePk(slice []ddl.IndexKey, s int) []ddl.IndexKey {
 	return append(slice[:s], slice[s+1:]...)
 }
 
-func RemoveFk(slice []ddl.Foreignkey, fkId string) []ddl.Foreignkey {
+func RemoveFk(slice []ddl.Foreignkey, fkId string, srcSchema schema.Table, tableId string) ([]ddl.Foreignkey, error) {
+	sessionState := session.GetSessionState()
+	tableIssues := sessionState.Conv.SchemaIssues[tableId].TableLevelIssues
+
 	pos := -1
 	for i, fk := range slice {
 		if fk.Id == fkId {
 			pos = i
+
+			// Remove foreign key action conversion warnings for dropped FK
+			//
+			// Issues ForeignKeyOnDelete and ForeignKeyOnUpdate might be added to SchemaIssues[tableId] multiple times
+			// due to existance of multiple FKs. We only delete one instance of these issues to maintain
+			// correct warnings for other FKs
+			srcFk, err := internal.GetSrcFkFromId(srcSchema.ForeignKeys, fk.Id)
```

Comment:
Are we adding multiple warnings with same message and no identifier in case of multiple unsupported FK actions?

Reply:
The warning body contains all the details of the foreign key and are not repeated, but internally the issue is added without this information so there's no way to differentiate